src/olympia/versions/models.py

```diff
@@ -215,6 +215,9 @@ class Meta(ModelBase.Meta):
             models.Index(fields=('addon',), name='addon_id'),
             models.Index(fields=('license',), name='license_id'),
         ]
+        constraints = [
+            models.UniqueConstraint(fields=('addon', 'version'), name='version_id'),
```

Comment:
I know this is WIP, but can we have a more descriptive name for this constraint? - it's got nothing to do with a version_id field. 

(the indexes above are also poorly named, but they are least match the column's name - i.e the index for `addon` is actually for `addon_id` in the database table, so the name matches)

Reply:
What do you suggest? I was looking at several other constraints defined in the code and they had similar or even "worse" naming (in terms of meaningfulness), so I went with something that aligns with what's being used in other places.

Reply:
Note that the name needs to be unique. I would suggest trying to follow the name `unique_together` would generate, which is `versions_addon_id_version_5a2e75b6_uniq`.

Reply:
dc9177812d